resources/public/api/conf/1.0/application.yaml

```diff
@@ -413,24 +439,110 @@ components:
     ExciseMovement:
       type: object
       required:
-        - movementId
         - consignorId
         - localReferenceNumber
         - status
       properties:
-        movementId:
-          type: string
-          format: uuid
         consignorId:
           $ref: '#/components/schemas/ExciseReferenceNumber'
         localReferenceNumber:
           $ref: '#/components/schemas/LocalReferenceNumber'
-        administrativeReferenceCode:
```

Comment:
I think we want to have this as part of a movement object, but it should be removed from the example on the submit draft movement as it won't have the ARC until the IE801 comes back